modules/dashboard/test/DashboardTest.php

```diff
@@ -90,7 +90,7 @@ function setUp()
              'CandID'       => '999888',
              'CenterID'     => '55',
              'UserID'       => '1',
-             'MRIQCStatus'  => 'Pass',
+             'MRIQCStatus'  => '',
```

Comment:
Travis couldn't pass one of the tests unless this was changed, because in the new query we're using `MRIQCStatus=''` now. I'm not sure why it was set to `Pass` in the first place during insertion...and I thought if it's a new scan, should this field already be set to `Pass`? Although I could be misunderstanding this field as I lack sufficient MRI knowledge.